### PR TITLE
136/ct as

### PR DESCRIPTION
**Pull request by ail3ngrimaldi**
Add commit with hidden keys

### CODE REVIEW

File: index.liquid [hunk 1]
```diff
@@ -205,9 +205,9 @@ questionButtons.forEach(button => {
 });
 
 // Airtable Connection
-const apiKeyAirtable = 'patNjZOfZHaFA6kkK.02d83b2cd73c1d2bd0e2ea8688f4fa29e6d640576d3125bb92e10f78e9cb830c'; //Do not upload like this!
-const baseId = 'app8afdeP1SxSAkKO';
-const table = 'tblsK7cdnFr55xP7W';
+const apiKeyAirtable = '{AIRTABLE_API_KEY}';
```
Comment (olanod) [inline index.liquid:208]: Since the entire file is a template perhaps we can use cobalt's native way to pass data to the file? (`{{ site.data.xxx }}`)

Comment (ail3ngrimaldi) [reply index.liquid:208]: If we pass the keys like that they should be on a gitignored file? is it possible to access them then on the script tag?

Comment (olanod) [reply index.liquid:208]: Sure, the keys shouldn't go into version control, still we can have a JSON data file with the variables defined without the actual value or with sample values similar to how `.env.example` files are included in version control but not the actual `.env` and then we generate the file during the deployment script.

Say there's a `_data/_env.json` file
```json
{
    "AIRTABLE_API_KEY": ""
}
```
A script later can update the value and create the file. e.g. `cat _data/env.json | jq '.AIRTABLE_API_KEY |= "secretvalue"' > _data/env.json`

Cobalt will have the variable available in the template `{{ site.data.env.AIRTABLE_API_KEY }}`

Comment (ail3ngrimaldi) [reply index.liquid]: I've added a new commit addressing the changes, but I have a few questions:

1. I couldn't get the variables to work when setting the env file with underscores (_). Is there a specific way to achieve this?
2. Does the choice between using vars or secrets depend on whether we set the keys using GitHub Actions or Cloudflare? In which case should we use each one?

**Review by olanod — APPROVED** [review]

**Pull request merged**